common/primitives/Cargo.toml

```diff
@@ -13,25 +13,25 @@ version = "0.0.0"
 targets = ['x86_64-unknown-linux-gnu']
 
 [dependencies]
-codec = { package = "parity-scale-codec", version = "3.6.1", default-features = false, features = [
+codec = { package = "parity-scale-codec", workspace = true, features = [
```

Comment:
I'm seeing warnings that package isn't needed when I try to build.

Reply:
Interestingly when I remove it fails to build. Ill take a look again.